### DISCUSSION TITLE
[IntelliSense] Add package.json field to detect configuration file in monorepos for V4

### DISCUSSION THREAD

**Discussion by lukemorales** [tailwindlabs/tailwindcss#13314] [Ideas]
Since V4 approach to configuring tailwind information relies on a css file, the VSCode extension is not able to find a configuration file for its intelissense in a monorepo setup. The only way I got it to work was to use experimental features to inform the extension where to look for the config:
```json
{
  "tailwindCSS.experimental.configFile": "./apps/web/src/app/globals.css"
}
```

It would be nice if we could add a field to `package.json` or another way to tell the extension where's the css file it needs to use for configuration

**Comment by JuerGenie** [comment]
As my solution, I wrote `.vscode/settings.json` in sub-package, just like this:

```js
// your-project/apps/web/.vscode/settings.json
{
  "tailwindCSS.experimental.configFile": "./globals.css"
}
```

You can apply this setting to each sub-project in your monorepo, and it should be effective.
It's work for me :D
<img width="1036" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/20407230/39b5a8ba-5a3c-4717-b3c3-2d156e281e32">

**Reply by george-zh-2023** [reply]
thank u, bruh)

**Reply by Hrobitaillie** [reply]
Thank you that dis the job !

**Comment by NaviTheCoderboi** [comment]
it doesn't works in a sveltekit project, i used this config
```json
{
	"editor.quickSuggestions": {
		"strings": "on"
	},
	"tailwindCSS.experimental.configFile": "./src/app.css"
}
```

**Reply by JuerGenie** [reply]
Not totally sure what‘s causing your issue :I

But for me, putting it under the SvelteKit project’s `.vscode` folder works fine too.
